testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -412,6 +412,107 @@ QUnit.test('From renders the right types of editors according to stylingMode opt
     assert.ok($testContainer.find('.dx-field-item .dx-textbox').hasClass('dx-editor-underlined'), 'right class rendered');
 });
 
+QUnit.module('Checkbox editor value', () => {
+    QUnit.test('FormData = {}', function(assert) {
+        const form = $('#form').dxForm({
+            items: [ { dataField: 'boolValue', editorType: 'dxCheckBox' } ],
+            formData: { }
+        }).dxForm('instance');
+
+        assert.equal(form.getEditor('boolValue').option('value'), false);
```

Comment:
I think we should sync the 'null or undefined' behavior for the supported data types and editors to avoid changes like these:
- https://github.com/DevExpress/DevExtreme/pull/8693/files#diff-6de1f7de416afb04d16b836719603a92L1771
- https://github.com/DevExpress/DevExtreme/pull/8693/files#diff-e8d63a47bb69f72a64f0ad5a966f3fe5L610
- https://github.com/DevExpress/DevExtreme/pull/8693/files#diff-e8d63a47bb69f72a64f0ad5a966f3fe5L617